hack/make.sh

```diff
@@ -83,7 +83,7 @@ if [ ! "$GOPATH" ]; then
 	exit 1
 fi
 
-if ${PKG_CONFIG} 'libsystemd' 2> /dev/null; then
+if [[ ${SYSTEMD:-1} == 1 ]] && ${PKG_CONFIG} 'libsystemd' 2> /dev/null; then
```

Comment:
Instead of the very generic `SYSTEMD` name here, can we use something more descriptive/specific?  I would suggest something like `NO_AUTO_BUILDTAG_JOURNALD` but open to other ideas/suggestions (especially opinions from other maintainers).  Also, `[[` isn't necessary here (`[` is sufficient).

```suggestion
if [ -z "${NO_AUTO_BUILDTAG_JOURNALD:-}" ] && ${PKG_CONFIG} 'libsystemd' 2> /dev/null; then
```

Reply:
> Also, `[[` isn't necessary here (`[` is sufficient).

The flip side is that the script explicitly uses bash, and the `[[` ***keyword*** has no downsides but allows bash to process it much earlier, in time to know that variables are variables -- the single `[` is a builtin program that is also available at `/usr/bin/[` and only knows that it received "argv arguments". This is why the single `[` needs you to quote variables that may be empty or contain spaces, while the `[[` allows omitting the quotes. In fact, even if you omit them by accident, it still works. :)

Reply:
Sure, this is more of a consistency request than an argument for one or the other (the rest of `hack/` minus three instances is consistently using bare `[`, which given we're in Bash, also doesn't shell out).